pkgs/tools/misc/hakuneko/default.nix

```diff
@@ -1,28 +1,89 @@
-{ stdenv, fetchurl, wxGTK30, openssl, curl }:
+{ atomEnv
+, autoPatchelfHook
+, dpkg
+, fetchurl
+, makeDesktopItem
+, makeWrapper
+, udev
+, stdenv
+, wrapGAppsHook
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+
+  throwSystem = throw "Unsupported system: ${system}";
 
-stdenv.mkDerivation rec {
   pname = "hakuneko";
-  version = "1.4.2";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/hakuneko/hakuneko_${version}_src.tar.gz";
-    sha256 = "76a63fa05e91b082cb5a70a8abacef005354e99978ff8b1369f7aa0af7615d52";
+  version = "5.0.8";
+
+  meta = with stdenv.lib; {
```

Comment:
meta is usually defined inside the derivation. There is no reason to define version outside either

Reply:
This was a leftover from simplenote, which I based this derivation of. Fixed.